### PR TITLE
Add CI for nbfmt

### PR DESCRIPTION
**Pull request by lamberta**
Auto-format notebooks using `nbfmt.py --ignore_warn`. This GitHub Action is triggered on push and will auto-add a commit by @tfdocsbot to the submitted pull request branch.

Tested here: https://github.com/lamberta/tf-docs-l10n/pull/4 ([actions log](https://github.com/lamberta/tf-docs-l10n/actions))

Will need to watch for how CLA bot reacts 🤖

Ref: https://github.com/tensorflow/docs-l10n/issues/128

### CODE REVIEW

**Comment by yashk2810** [comment]
Is it possible to whitelist tfdocsbot with the CLA not?

**Comment by lamberta** [comment]
> Is it possible to whitelist tfdocsbot with the CLA bot?

I think it already is (b/130115716), I just haven't tested (and using a slightly different email here)

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+name: CI
+on: push
+
+jobs:
+  format_notebooks:
+    name: Format notebooks
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/setup-python@v1
```
[the remark this answers is not in the document]
Comment (lamberta) [reply .github/workflows/ci.yaml]: it is. 3.8

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        pip install --upgrade absl-py
+        cd ./docs-l10n && git fetch origin master:master
+        git config --global user.email "tfdocsbot+no-reply@gmail.com"
+        git config --global user.name "TF DocsBot"
+
+    - name: nbfmt
+      run: |
+        cd ./docs-l10n
+        echo "Run nbfmt.py on updated notebooks:"
+        NOTEBOOKS_CHANGED="$(git diff --name-only master | grep '\.ipynb$')"
```
[the remark this answers is not in the document]
Comment (lamberta) [reply .github/workflows/ci.yaml]: ok, yeah, looked into it. `merge-base` fetches the last common commit to compare against.
this diff compares the entire (pull request) branch against master---which should work in this case (or at least has been in tests). Will keep and eye out, tho

**Comment by lamberta** [comment]
Thanks. Will wait until tomorrow to merge so I don't get flooded with emails in the morning in case it doesn't work as expected.

**Pull request merged**